casper/src/main/scala/coop/rchain/casper/util/EventConverter.scala

```diff
@@ -13,34 +13,39 @@ import coop.rchain.rspace.trace.{
 import scala.collection.immutable.Seq
 
 object EventConverter {
+  implicit def byteStringToBlake2b256Hash(hash: ByteString): Blake2b256Hash =
+    Blake2b256Hash.fromByteArray(hash.toByteArray)
+
+  implicit def blake2b256HashToByteString(hash: Blake2b256Hash): ByteString =
+    ByteString.copyFrom(hash.bytes.toArray)
+
   def toCasperEvent(event: RspaceEvent): Event = event match {
-    case RspaceProduce(produce) =>
-      Event(Produce(ProduceEvent(ByteString.copyFrom(produce.bytes.toArray))))
-    case RspaceConsume(consume) =>
-      Event(Consume(ConsumeEvent(ByteString.copyFrom(consume.bytes.toArray))))
+    case produce: RspaceProduce =>
```

Comment:
Just wanted to make it seamless with lines below, where we use more complicated patterns. Since compiler/casper teams work with this code much more often - your opinion has top priority. Please let me know if you prefer this style here.

Reply:
For me it's fine as it is.